WickedEngine/wiGraphics.h

```diff
@@ -469,6 +469,22 @@ namespace wiGraphics
 		PRIMITIVETOPOLOGY			pt = TRIANGLELIST;
 		uint32_t					sampleMask = 0xFFFFFFFF;
 	};
+	struct Memory
```

Comment:
I'd prefer these structs declared within the GPUBarrier struct

Reply:
Done! Just for future reference, you can't declare a struct like `struct Memory { ... } memory;` in a union. Instead, use `struct { ... } memory;' or define them seperetly.

Reply:
That's a little bummer, but no big deal. :(